ecosystem/sep-0031.md

```diff
@@ -119,31 +122,24 @@ The response should be a JSON object like:
          "fee_percent":1,
          "min_amount":0.1,
          "max_amount":1000,
+         "sender_sep12_type": "sep31-sender",
+         "receiver_sep12_type": "sep31-receiver",
          "fields":{
-            "sender":{
-               "first_name": {
-                  "description": "The sender's first name"
-               },
-               "last_name": {
-                  "description": "The sender's last name"
+            "sender": {
+               "id": {
+                  "description": "The id returned in the SEP-12 PUT /customer response for the sender",
```

Comment:
I'm not sure i like having the sender and receiver objects here like this, it implies that you can have other fields.  It seems like we're strongly opinionated that all kyc fields go through sep12 so we may want to remove these here and just prescribe that you'll add sender_id and receiver_id to the /send call

Reply:
I agree, I'll make the change.